notifications_utils/recipients.py

```diff
@@ -501,7 +501,6 @@ def validate_recipient(recipient, template_type, international_sms=False):
     return {
         'email': validate_email_address,
         'sms': partial(validate_phone_number, international=international_sms),
-        'letter': lambda _: recipient,
```

Comment:
Check we aren't still calling this case elsewhere

Reply:
All uses in this repo:
![image](https://user-images.githubusercontent.com/355079/80356001-bbff3200-8870-11ea-8565-82986e17d2df.png)

API:
![image](https://user-images.githubusercontent.com/355079/80356516-7858f800-8871-11ea-9414-6058e6940ca1.png)

Admin:
![image](https://user-images.githubusercontent.com/355079/80356540-7f800600-8871-11ea-9cbd-4a47011a47a1.png)